EventFilter/GEMRawToDigi/plugins/GEMDigiToRawModule.cc

```diff
@@ -106,6 +118,12 @@ void GEMDigiToRawModule::produce(edm::StreamID iID, edm::Event& iEvent, edm::Eve
     const GEMDigiCollection::Range& digis = etaPart.second;
     for (auto digi = digis.first; digi != digis.second; ++digi) {
       int bx = digi->bx();
+      if (simulatePulseStretching_) {
+        if (bx < minBunch_ or bx > maxBunch_)
```

Comment:
Wouldn't it be more compact to just write
`if( bx > minbunch_  && bx < maxBunch_) bx =0` 
?
Then you don't need the continue statement

Reply:
ah, yes, you're right.  I miscounted the brackets.  There's indeed more functional code below. Thanks